### PR TITLE
Add rowcount property for Python cursor objects

### PR DESCRIPTION
**Pull request by hannes**
as the law has commanded

### CODE REVIEW

File: tools/pythonpkg/tests/fast/api/test_dbapi10.py [hunk 1]
```diff
@@ -25,3 +25,8 @@ def test_description(self, query, column_name, string_type, real_type, duckdb_cu
 
     def test_none_description(self, duckdb_empty_cursor):
         assert duckdb_empty_cursor.description is None
+
+
+class TestCursorRowcount(object):
+    def test_fetchall_date(self, duckdb_cursor):
```
Comment (Mause) [inline tools/pythonpkg/tests/fast/api/test_dbapi10.py:31]:
```suggestion
    def test_rowcount(self, duckdb_cursor):
```

**Review by Mause — APPROVED** [review]
Hah, that'll do it yeah 😁

**Review by Tishj — APPROVED** [review]
Looks good to me, we can extend this with actual logic whenever, for now just having the method reduces the threshold for that 👍

File: tools/pythonpkg/tests/fast/api/test_dbapi10.py (hunk 1 above)
Comment (hannes) [reply tools/pythonpkg/tests/fast/api/test_dbapi10.py]: Thanks

**Comment by Tishj** [comment]
Ah I think you ran `regenerate_python_stubs.sh`, which should probably be deleted actually
It breaks all the manual changes made, instead we just modify the stubs manually

**Pull request merged**